src/components/forms/DateField/utils.js

```diff
@@ -47,7 +47,7 @@ export const getDateLocaleMeta = locale => {
 };
 
 export const convertMonth = (month, toAdd) => {
-  if (!month) return '';
+  if (month === '' || month == null) return '';
   const monthNumber = parseInt(month);
   return String(monthNumber + toAdd);
```

Comment:
```suggestion
  const monthNumber = parseInt(month);
  return isNaN(monthNumber)?'':String(monthNumber + toAdd);
```

This catches `undefined` and whatever JS idiosyncrasies  I haven't thought of.
Is it okay for this to go out of bounds to 13? :thinking: 12?

Reply:
Undefined is already caught with the non-strict equal null. This is ugly but idiomatic javascript.

13 and the likes should be covered later when a date instance is constructed and checked against isNaN